underscore.js

```diff
@@ -647,6 +647,20 @@
     return range;
   };
 
+  // Split an **array** into several arrays containing **count** or less elements
+  // of initial array
+  _.chunk = function(array, count) {
+    var result = [];
+
+    if (array == null || count <= 0) return [];
+
+    for (var i = 0, length = array.length; i < length;) {
```

Comment:
Mind using a while-loop? Every other for-loop in the codebase has an increment expression...